src/utils/urls.js

```diff
@@ -72,6 +72,28 @@ function createUrl (path, websockets = false, baseOnly = false) {
   return normalize(url)
 }
 
+/**
+ * Get the user name from the current browser URL. If the `window.location.pathname`
+ * object is not available (e.g. running without a browser?) then the
+ * fallback given value is returned.
+ *
+ * The expected URL format is <protocol>://<hostname>:<port>/user/<username>/. The returned
+ * value will be either the <username> or the given fallback username.
+ *
+ * @param {string} fallbackValue - fallback value
+ * @returns {string} - the
+ */
+function getUserNameFromUrl (fallbackValue) {
+  return window.location.pathname
+    ? window.location.pathname
```

Comment:
TodayILearned it's better/safer to use `window.location` than `document.location`, even though I get the same with both on Firefox/Chromium/etc: https://stackoverflow.com/a/2431375